pldoc/static/sass/_components.scss

```diff
@@ -342,6 +364,209 @@
 // color swatches
 .pldoc-pattern-colors {
 
+    .brand-primary {
```

Comment:
Good question, the classes are just for the actual rendered color palette in http://ux.edx.org/elements/colors/. These are not meant to be used as official classes that the UXPL bower package supports in a proj - hence why I'm moving all of this Sass/CSS to be part of the Documentation Site and not the package.

Seem still legit?